src/screens/ContentSelector/index.js

```diff
@@ -1,226 +1,143 @@
-import React, { Component } from 'react'
-import List from './SelectionList'
-import Button from 'components/Button'
-import Icon from 'components/Icon'
-import { css } from 'glamor'
-
-const scrollSectionStyle = css({
-  height: '300px',
-  maxHeight: '300px',
-  overflowY: 'auto',
-})
-
-const lastChildStyle = css({
-  '&:last-child': {
-    borderBottomWidth: '0px',
-  },
-})
-
-const inputStyle = css({
-  height: '2.2em',
-  textIndent: '1em',
-  '::placeholder': {
-    color: '#aaa',
-    alignItem: 'center',
-    display: 'flex',
-  },
-})
-
-export default class LessonsSelector extends Component {
-
-  componentWillMount = () => {
-    this.setState({
-      // update this to the relevant sytarting array elements
-      lessonList: [
-        'akash',
-        'vojta',
-        'evgeniy',
-        'joel',
-        'taylor',
-        'raquel',
-        'pete keen',
-      ],
-      defaultList: [
-        'asdfja',
-        'vojsadfta',
-        'evgasdfasdeniy',
-      ],
-      allSelected: false,
-    })
-  }
+import React from 'react'
+import {some, remove, isEqual, debounce, isUndefined} from 'lodash'
+import {DragDropContext, Draggable, Droppable} from 'react-beautiful-dnd'
+import selectedLessons from './data/selectedLessons'
+import allLessons from './data/lessons'
+import axios from 'axios'
+
+//this is actually a component
+
+const http = axios.create() //mobx
 
+export default class ContentSelector extends React.Component {
```

Comment:
backed out of the render props extravaganza.

Reply:
So. The instructor can drag and drop from left column to the right column, is it?

Also, I think we can have explicit `addAll` and `removeAll` function that can be used.

Reply:
We don’t need to add all or remove all. The dragging is on the left side, because it can be a sortable list.